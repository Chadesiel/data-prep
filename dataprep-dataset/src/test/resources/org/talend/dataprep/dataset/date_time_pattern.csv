id;date
1;12/12/2005 12:50:49
2;01/01/1980 11:07:50
3;10/11/2003 10:06:25
4;02/02/2050 09:08:21
5;01/08/1890 08:08:07
