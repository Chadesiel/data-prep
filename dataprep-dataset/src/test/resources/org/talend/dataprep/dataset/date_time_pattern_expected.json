{
  "count": 5,
  "valid": 5,
  "invalid": 0,
  "empty": 0,
  "max": "NaN",
  "min": "NaN",
  "mean": "NaN",
  "variance": "NaN",
  "duplicateCount": 0,
  "distinctCount": 5,
  "frequencyTable": [
    {
      "data": "12/12/2005 12:50:49",
      "occurrences": 1
    },
    {
      "data": "02/02/2050 09:08:21",
      "occurrences": 1
    },
    {
      "data": "10/11/2003 10:06:25",
      "occurrences": 1
    },
    {
      "data": "01/01/1980 11:07:50",
      "occurrences": 1
    },
    {
      "data": "01/08/1890 08:08:07",
      "occurrences": 1
    }
  ],
  "patternFrequencyTable": [
    {
      "pattern": "d/M/yyyy H:m:s",
      "occurrences": 5
    }
  ],
  "quantiles": {
    "median": "NaN",
    "lowerQuantile": "NaN",
    "upperQuantile": "NaN"
  },
  "histogram": {
    "type": "date",
    "items": [
      {
        "occurrences": 1,
        "range": {
          "min": {
            "year": 1890,
            "month": "JANUARY",
            "chronology": {
              "calendarType": "iso8601",
              "id": "ISO"
            },
            "dayOfMonth": 1,
            "dayOfWeek": "WEDNESDAY",
            "era": "CE",
            "dayOfYear": 1,
            "leapYear": false,
            "monthValue": 1
          },
          "max": {
            "year": 1900,
            "month": "JANUARY",
            "chronology": {
              "calendarType": "iso8601",
              "id": "ISO"
            },
            "dayOfMonth": 1,
            "dayOfWeek": "MONDAY",
            "era": "CE",
            "dayOfYear": 1,
            "leapYear": false,
            "monthValue": 1
          }
        }
      },
      {
        "occurrences": 0,
        "range": {
          "min": {
            "year": 1900,
            "month": "JANUARY",
            "chronology": {
              "calendarType": "iso8601",
              "id": "ISO"
            },
            "dayOfMonth": 1,
            "dayOfWeek": "MONDAY",
            "era": "CE",
            "dayOfYear": 1,
            "leapYear": false,
            "monthValue": 1
          },
          "max": {
            "year": 1910,
            "month": "JANUARY",
            "chronology": {
              "calendarType": "iso8601",
              "id": "ISO"
            },
            "dayOfMonth": 1,
            "dayOfWeek": "SATURDAY",
            "era": "CE",
            "dayOfYear": 1,
            "leapYear": false,
            "monthValue": 1
          }
        }
      },
      {
        "occurrences": 0,
        "range": {
          "min": {
            "year": 1910,
            "month": "JANUARY",
            "chronology": {
              "calendarType": "iso8601",
              "id": "ISO"
            },
            "dayOfMonth": 1,
            "dayOfWeek": "SATURDAY",
            "era": "CE",
            "dayOfYear": 1,
            "leapYear": false,
            "monthValue": 1
          },
          "max": {
            "year": 1920,
            "month": "JANUARY",
            "chronology": {
              "calendarType": "iso8601",
              "id": "ISO"
            },
            "dayOfMonth": 1,
            "dayOfWeek": "THURSDAY",
            "era": "CE",
            "dayOfYear": 1,
            "leapYear": true,
            "monthValue": 1
          }
        }
      },
      {
        "occurrences": 0,
        "range": {
          "min": {
            "year": 1920,
            "month": "JANUARY",
            "chronology": {
              "calendarType": "iso8601",
              "id": "ISO"
            },
            "dayOfMonth": 1,
            "dayOfWeek": "THURSDAY",
            "era": "CE",
            "dayOfYear": 1,
            "leapYear": true,
            "monthValue": 1
          },
          "max": {
            "year": 1930,
            "month": "JANUARY",
            "chronology": {
              "calendarType": "iso8601",
              "id": "ISO"
            },
            "dayOfMonth": 1,
            "dayOfWeek": "WEDNESDAY",
            "era": "CE",
            "dayOfYear": 1,
            "leapYear": false,
            "monthValue": 1
          }
        }
      },
      {
        "occurrences": 0,
        "range": {
          "min": {
            "year": 1930,
            "month": "JANUARY",
            "chronology": {
              "calendarType": "iso8601",
              "id": "ISO"
            },
            "dayOfMonth": 1,
            "dayOfWeek": "WEDNESDAY",
            "era": "CE",
            "dayOfYear": 1,
            "leapYear": false,
            "monthValue": 1
          },
          "max": {
            "year": 1940,
            "month": "JANUARY",
            "chronology": {
              "calendarType": "iso8601",
              "id": "ISO"
            },
            "dayOfMonth": 1,
            "dayOfWeek": "MONDAY",
            "era": "CE",
            "dayOfYear": 1,
            "leapYear": true,
            "monthValue": 1
          }
        }
      },
      {
        "occurrences": 0,
        "range": {
          "min": {
            "year": 1940,
            "month": "JANUARY",
            "chronology": {
              "calendarType": "iso8601",
              "id": "ISO"
            },
            "dayOfMonth": 1,
            "dayOfWeek": "MONDAY",
            "era": "CE",
            "dayOfYear": 1,
            "leapYear": true,
            "monthValue": 1
          },
          "max": {
            "year": 1950,
            "month": "JANUARY",
            "chronology": {
              "calendarType": "iso8601",
              "id": "ISO"
            },
            "dayOfMonth": 1,
            "dayOfWeek": "SUNDAY",
            "era": "CE",
            "dayOfYear": 1,
            "leapYear": false,
            "monthValue": 1
          }
        }
      },
      {
        "occurrences": 0,
        "range": {
          "min": {
            "year": 1950,
            "month": "JANUARY",
            "chronology": {
              "calendarType": "iso8601",
              "id": "ISO"
            },
            "dayOfMonth": 1,
            "dayOfWeek": "SUNDAY",
            "era": "CE",
            "dayOfYear": 1,
            "leapYear": false,
            "monthValue": 1
          },
          "max": {
            "year": 1960,
            "month": "JANUARY",
            "chronology": {
              "calendarType": "iso8601",
              "id": "ISO"
            },
            "dayOfMonth": 1,
            "dayOfWeek": "FRIDAY",
            "era": "CE",
            "dayOfYear": 1,
            "leapYear": true,
            "monthValue": 1
          }
        }
      },
      {
        "occurrences": 0,
        "range": {
          "min": {
            "year": 1960,
            "month": "JANUARY",
            "chronology": {
              "calendarType": "iso8601",
              "id": "ISO"
            },
            "dayOfMonth": 1,
            "dayOfWeek": "FRIDAY",
            "era": "CE",
            "dayOfYear": 1,
            "leapYear": true,
            "monthValue": 1
          },
          "max": {
            "year": 1970,
            "month": "JANUARY",
            "chronology": {
              "calendarType": "iso8601",
              "id": "ISO"
            },
            "dayOfMonth": 1,
            "dayOfWeek": "THURSDAY",
            "era": "CE",
            "dayOfYear": 1,
            "leapYear": false,
            "monthValue": 1
          }
        }
      },
      {
        "occurrences": 0,
        "range": {
          "min": {
            "year": 1970,
            "month": "JANUARY",
            "chronology": {
              "calendarType": "iso8601",
              "id": "ISO"
            },
            "dayOfMonth": 1,
            "dayOfWeek": "THURSDAY",
            "era": "CE",
            "dayOfYear": 1,
            "leapYear": false,
            "monthValue": 1
          },
          "max": {
            "year": 1980,
            "month": "JANUARY",
            "chronology": {
              "calendarType": "iso8601",
              "id": "ISO"
            },
            "dayOfMonth": 1,
            "dayOfWeek": "TUESDAY",
            "era": "CE",
            "dayOfYear": 1,
            "leapYear": true,
            "monthValue": 1
          }
        }
      },
      {
        "occurrences": 1,
        "range": {
          "min": {
            "year": 1980,
            "month": "JANUARY",
            "chronology": {
              "calendarType": "iso8601",
              "id": "ISO"
            },
            "dayOfMonth": 1,
            "dayOfWeek": "TUESDAY",
            "era": "CE",
            "dayOfYear": 1,
            "leapYear": true,
            "monthValue": 1
          },
          "max": {
            "year": 1990,
            "month": "JANUARY",
            "chronology": {
              "calendarType": "iso8601",
              "id": "ISO"
            },
            "dayOfMonth": 1,
            "dayOfWeek": "MONDAY",
            "era": "CE",
            "dayOfYear": 1,
            "leapYear": false,
            "monthValue": 1
          }
        }
      },
      {
        "occurrences": 0,
        "range": {
          "min": {
            "year": 1990,
            "month": "JANUARY",
            "chronology": {
              "calendarType": "iso8601",
              "id": "ISO"
            },
            "dayOfMonth": 1,
            "dayOfWeek": "MONDAY",
            "era": "CE",
            "dayOfYear": 1,
            "leapYear": false,
            "monthValue": 1
          },
          "max": {
            "year": 2000,
            "month": "JANUARY",
            "chronology": {
              "calendarType": "iso8601",
              "id": "ISO"
            },
            "dayOfMonth": 1,
            "dayOfWeek": "SATURDAY",
            "era": "CE",
            "dayOfYear": 1,
            "leapYear": true,
            "monthValue": 1
          }
        }
      },
      {
        "occurrences": 2,
        "range": {
          "min": {
            "year": 2000,
            "month": "JANUARY",
            "chronology": {
              "calendarType": "iso8601",
              "id": "ISO"
            },
            "dayOfMonth": 1,
            "dayOfWeek": "SATURDAY",
            "era": "CE",
            "dayOfYear": 1,
            "leapYear": true,
            "monthValue": 1
          },
          "max": {
            "year": 2010,
            "month": "JANUARY",
            "chronology": {
              "calendarType": "iso8601",
              "id": "ISO"
            },
            "dayOfMonth": 1,
            "dayOfWeek": "FRIDAY",
            "era": "CE",
            "dayOfYear": 1,
            "leapYear": false,
            "monthValue": 1
          }
        }
      },
      {
        "occurrences": 0,
        "range": {
          "min": {
            "year": 2010,
            "month": "JANUARY",
            "chronology": {
              "calendarType": "iso8601",
              "id": "ISO"
            },
            "dayOfMonth": 1,
            "dayOfWeek": "FRIDAY",
            "era": "CE",
            "dayOfYear": 1,
            "leapYear": false,
            "monthValue": 1
          },
          "max": {
            "year": 2020,
            "month": "JANUARY",
            "chronology": {
              "calendarType": "iso8601",
              "id": "ISO"
            },
            "dayOfMonth": 1,
            "dayOfWeek": "WEDNESDAY",
            "era": "CE",
            "dayOfYear": 1,
            "leapYear": true,
            "monthValue": 1
          }
        }
      },
      {
        "occurrences": 0,
        "range": {
          "min": {
            "year": 2020,
            "month": "JANUARY",
            "chronology": {
              "calendarType": "iso8601",
              "id": "ISO"
            },
            "dayOfMonth": 1,
            "dayOfWeek": "WEDNESDAY",
            "era": "CE",
            "dayOfYear": 1,
            "leapYear": true,
            "monthValue": 1
          },
          "max": {
            "year": 2030,
            "month": "JANUARY",
            "chronology": {
              "calendarType": "iso8601",
              "id": "ISO"
            },
            "dayOfMonth": 1,
            "dayOfWeek": "TUESDAY",
            "era": "CE",
            "dayOfYear": 1,
            "leapYear": false,
            "monthValue": 1
          }
        }
      },
      {
        "occurrences": 0,
        "range": {
          "min": {
            "year": 2030,
            "month": "JANUARY",
            "chronology": {
              "calendarType": "iso8601",
              "id": "ISO"
            },
            "dayOfMonth": 1,
            "dayOfWeek": "TUESDAY",
            "era": "CE",
            "dayOfYear": 1,
            "leapYear": false,
            "monthValue": 1
          },
          "max": {
            "year": 2040,
            "month": "JANUARY",
            "chronology": {
              "calendarType": "iso8601",
              "id": "ISO"
            },
            "dayOfMonth": 1,
            "dayOfWeek": "SUNDAY",
            "era": "CE",
            "dayOfYear": 1,
            "leapYear": true,
            "monthValue": 1
          }
        }
      },
      {
        "occurrences": 0,
        "range": {
          "min": {
            "year": 2040,
            "month": "JANUARY",
            "chronology": {
              "calendarType": "iso8601",
              "id": "ISO"
            },
            "dayOfMonth": 1,
            "dayOfWeek": "SUNDAY",
            "era": "CE",
            "dayOfYear": 1,
            "leapYear": true,
            "monthValue": 1
          },
          "max": {
            "year": 2050,
            "month": "JANUARY",
            "chronology": {
              "calendarType": "iso8601",
              "id": "ISO"
            },
            "dayOfMonth": 1,
            "dayOfWeek": "SATURDAY",
            "era": "CE",
            "dayOfYear": 1,
            "leapYear": false,
            "monthValue": 1
          }
        }
      },
      {
        "occurrences": 1,
        "range": {
          "min": {
            "year": 2050,
            "month": "JANUARY",
            "chronology": {
              "calendarType": "iso8601",
              "id": "ISO"
            },
            "dayOfMonth": 1,
            "dayOfWeek": "SATURDAY",
            "era": "CE",
            "dayOfYear": 1,
            "leapYear": false,
            "monthValue": 1
          },
          "max": {
            "year": 2060,
            "month": "JANUARY",
            "chronology": {
              "calendarType": "iso8601",
              "id": "ISO"
            },
            "dayOfMonth": 1,
            "dayOfWeek": "THURSDAY",
            "era": "CE",
            "dayOfYear": 1,
            "leapYear": true,
            "monthValue": 1
          }
        }
      }
    ]
  },
  "textLengthSummary": {
    "minimalLength": "NaN",
    "maximalLength": "NaN",
    "averageLength": "NaN"
  }
}